package.json

```diff
@@ -1,4 +1,6 @@
 {
+  "name": "porta",
+  "version": "0.0.1",
```

Comment:
Why no setting the actual porta version `2.13.2`?

Reply:
@jlledom is that the porta version, or the 3scale version ?

Reply:
(but, yes, we absolutely need to set the correct version here)

Reply:
@gsaslis is there a porta version different from 3scale version?

Reply:
well: 

- latest 3scale product build is 2.13.4 (not 2.13.2) 
- the porta component can have a different version number than the whole product does
- we **do** use a different version for the porta (system) build - (see tags in https://catalog.redhat.com/software/containers/3scale-amp2/system-rhel7/5d80b9d7d70cc5521d516a49 )

Having said that, this here is the community porta (so a different versioning policy is 100% fine) **and** (and this is important) I don't think this `name` + `version` actually makes much difference to porta itself _anyway_ (otherwise I'd have expected to find it here already)...

tl;dr 

`42` would also work. 🙃

Reply:
maybe instead of `0.0.1` use `placeholder` or some other value indicating that it is not supposed to correspond to actual version

Reply:
The version must be in semantic versioning format according to nodejs docs, so what do you guys think would be a suitable "not a version" version? Personally, I don't see how any other version will be any better than 0.0.1, although none of them are ideal...

Reply:
idk, maybe `0.0.0`, otherwise `0.0.1` looks second bet option in this case :)

Reply:
I've set it to 0.0.0 :)